src/server/graphql/models/Team/teamMutation.js

```diff
@@ -41,9 +86,9 @@ export default {
         meetingId: shortid.generate(),
         activeFacilitator: facilitatorId,
         facilitatorPhase: CHECKIN,
-        facilitatorPhaseItem: 0,
+        facilitatorPhaseItem: '0',
```

Comment:
always treat the `phaseItem` as a string. This is because the agenda round will use an `agendaItemId`. We do that because agenda items can be moved around & when we enable drag n drop, we'll probably make the order a double instead of an integer so we only need to call 1 update, not `n`.